.github/ISSUE_TEMPLATE/packaging_request.yml

```diff
@@ -0,0 +1,52 @@
+name: Packaging request
+description: A project is not yet available in Nixpkgs.
+labels: '0.kind: packaging request'
+
+body:
+  - type: input
+    attributes:
+      label: Project name
+      description: Name of the project which you would like to request a package for.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Project description
+      description: Provide a short description of the project.
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Source URL
+      description: Where the source code of the project can be found.
+      placeholder: https://github.com/octocat/Hello-World/
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Homepage URL
+      description: Homepage of the project. Can be its source repository if the project does not have a dedicated homepage.
+      placeholder: https://example.com/
+    validations:
+      required: true
```

Comment:
Often times source code == home page. No need to duplicate that.

Reply:
I don't even think we need this field at all tbh. The home page will contain a link to the source, so that can also be put in "Source URL".

Reply:
I agree. Either a link to the source code or the homepage would be sufficient.

I made two separate fields simply because that was what was in the old template.